docs/en-US/component/time-picker.md

```diff
@@ -76,7 +76,9 @@ time-picker/range
 
 ## Methods
 
-| Method | Description               | Parameters |
-| ------ | ------------------------- | ---------- |
-| focus  | focus the Input component | —          |
-| blur   | blur the Input component  | —          |
+| Method      | Description                  | Parameters |
+| ----------- | ---------------------------- | ---------- |
+| focus       | focus the Input component    | —          |
+| blur        | blur the Input component     | —          |
+| handleOpen  | opens the TimePicker popper  | —          |
+| handleClose | closes the TimePicker popper | —          |
```

Comment:
use `close` instead of `closes`

Reply:
Completely agree 👌
I have just committed a fix for this.

Reply:
nice